solr/solrj/src/test/org/apache/solr/client/solrj/LargeVolumeTestBase.java

```diff
@@ -41,7 +41,7 @@ public abstract class LargeVolumeTestBase extends EmbeddedSolrServerTestBase {
 
   @Test
   public void testMultiThreaded() throws Exception {
-    SolrClient client = this.getSolrClient();
+    SolrClient client = getSolrClient();
```

Comment:
Should this be `solrClientTestRule.getSolrClient()` ?   When do we want to use the explicit `solrClientTestRule` and when do we have a helper like `getSolrClient()`.   In some ways, I like the explicit verbosity of the `solrClientTestRule.getSolrClient()` telling me where the client came from...

Reply:
`getSolrClient()` in `EmbeddedSolrServerTestBase` returns `solrClientTestRule.getSolrClient()`

Reply:
I get that..   What I'm wondering is that it appears that sometimes we use the helper pattern of a `getSolrClient()`, but in other tests we use the full explicit `solrClientTestRule.getSolrClient()`....    Feels like we should try and have one pattern?   Maybe just a bikeshed..

Reply:
EmbeddedSolrServerTestBase is henceforth deprecated; it needn't exist; it barely does anything.  If someone wishes to remove it, then they can make that change then.  But those tests use this test base today so... why change them today.

Reply:
Right...   I think you suggested that...   So let's make sure to create a JIRA post merging this to get EmbeddedSolrServerTestBase removed ;-)   I'd be happy to sign up for that, removing deprecations is fun!